### ISSUE TITLE
Unknown inspection: SqlCheckUsingColumns

### ISSUE THREAD

**Issue by ArtemGoutsoul** [kalessil/phpinspectionsea#1420]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) update v3.0.16.2 |
| Language level | PHP 7.0                         |

Current behaviour
--
```
<?php

/** @noinspection SqlCheckUsingColumns */
```
Produces: Unknown inspection: SqlCheckUsingColumns

Expected behaviour: should not produce the warning
--

Environment details
---
PhpStorm 2019.3 EAP
Build #PS-193.3793.21, built on September 26, 2019
PhpStorm EAP User
Expiration date: October 26, 2019
Runtime version: 11.0.4+10-b485.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1450M
Cores: 12
Registry: run.processes.with.pty=TRUE, debugger.watches.in.variables=false, ide.tooltip.initialDelay=187
Non-Bundled Plugins: CMD Support, Jetbrains TeamCity Plugin, NEON support, Scratch, Statistic, String Manipulation, com.dmarcotte.handlebars, com.intellij.ideolog, com.jetbrains.lang.ejs, com.kalessil.phpStorm.phpInspectionsEA, lv.midiana.misc.phpstorm-plugins.deep-keys, lv.midiana.misc.idea-plugins.deep-js-completion, net.seesharpsoft.intellij.plugins.csv, org.intellij.tasks.navigation, org.psliwa.idea.composer, ru.adelf.idea.dotenv, some.awesome

For the sake of completeness: I first thought this was a PhpStorm issue, so I filed a bug there: https://youtrack.jetbrains.com/issue/WI-47360

**Comment by kalessil** [comment]
Fixed!
